src/python/pants/rules/core/list_target_types.py

```diff
@@ -43,10 +43,20 @@ class AbbreviatedTargetInfo:
     def create(cls, target_type: Type[Target]) -> "AbbreviatedTargetInfo":
         return cls(alias=target_type.alias, description=get_docstring_summary(target_type))
 
-    def format_for_cli(self, *, console: Console, longest_target_alias: int) -> str:
-        formatted_alias = console.cyan(f"{self.alias:>{longest_target_alias + 1}}:")
-        description = self.description or "<no description>"
-        return f"{formatted_alias} {description}"
+    def format_for_cli(self, console: Console, *, longest_target_alias: int) -> str:
+        chars_before_description = longest_target_alias + 2
+        alias = console.cyan(f"{self.alias}".ljust(chars_before_description))
+        if not self.description:
+            description = "<no description>"
```

Comment:
Not sure I love this. Maybe empty would look more user friendly. Here in a way it feels like punishing the user of the tool for the developer not taking the time to provide a description.

Reply:
Sgtm, I agree.

Reply:
Oh, hm, @gshuflin explicitly added this feature in https://github.com/pantsbuild/pants/pull/8088 and I remember it confused him why something was blank, e.g. wondering if it was a bug in Pants.

I would rather that we place a higher burden on us Pants devs to ensure we're writing good generated documentation. For example, when adding a new goal, we should expect Pants devs to put a screenshot of how it renders in `./pants goals`.

One benefit of `<no description>` is that it's uglier than a blank line, so it motivates us Pants devs to do something. Specifically because of seeing `<no description>`, I've added docstring to two goals in the past two months. I probably would have done nothing if it were just an empty line.